app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt

```diff
@@ -96,15 +96,26 @@ class HomeMenu(
         }
     }
 
-    private fun getSyncItemTitle(): String =
-        accountManager.accountProfileEmail ?: context.getString(R.string.sync_menu_sign_in)
+    private fun syncSignInMenuItem(): BrowserMenuImageText? {
+        val syncItemTitle =
+            if (context.components.backgroundServices.accountManagerAvailableQueue.isReady()) {
+                accountManager.accountProfileEmail ?: context.getString(R.string.sync_menu_sign_in)
+            } else {
+                null
+            }
 
-    private val syncSignInMenuItem = BrowserMenuImageText(
-        getSyncItemTitle(),
-        R.drawable.ic_synced_tabs,
-        primaryTextColor
-    ) {
-        onItemTapped.invoke(Item.SyncAccount(accountManager.accountState))
+        return when (syncItemTitle) {
+            null -> null
+            else -> {
+                BrowserMenuImageText(
+                    syncItemTitle,
+                    R.drawable.ic_signed_out,
```

Comment:
I noticed that the icon doesn't match the toolbar menu. I've updated the icon to ic_signed_out here but would require confirmation from UX.